File: code_samples/dart/database_query_builder.dart
// Database: Query Builder
// AI/ML Training Sample

class QueryBuilder {
  String _data = '';
  
  void process(String input) {
    _data = input;
  }
  
  String getData() => _data;
  
  bool validate() => _data.isNotEmpty;
}

void main() {
  final instance = QueryBuilder();
  instance.process('example');
  print('Data: ${instance.getData()}');
  print('Valid: ${instance.validate()}');
}

File: code_samples/kotlin/database_query_builder.kt
/**
 * Database: Query Builder
 * AI/ML Training Sample
 */

class QueryBuilder {
    var data: String = ""
        private set
    
    fun process(input: String) {
        data = input
    }
    
    fun getData(): String = data
    
    fun validate(): Boolean = data.isNotEmpty()
}

fun main() {
    val instance = QueryBuilder()
    instance.process("example")
    println("Data: ${instance.getData()}")
    println("Valid: ${instance.validate()}")
}
